test/README.md

```diff
@@ -175,13 +175,15 @@ In #99, I've added a _file_test function (to match the existing _binary_test) th
                     .....
                     .....
                     .....
-    @unittest.skipUnless(int(os.getenv("LONG_TESTS")) > 0, "Skipping long tests")
+    @unittest.skipUnless(LONG_TESTS() > 0, "Skipping long tests")
```

Comment:
We could do something like this too (get from `os.environ` and provide default of 0 if it doesn't exist):

```suggestion
    @unittest.skipUnless(int(os.environ.get("LONG_TESTS", 0)) > 0, "Skipping long tests")
```

But I do like the idea of having it be a function, since then we only have to change the logic in one place if we decide to incorporate other checks

Reply:
the problem was that int(os.environ.get()) fails if LONG_TESTS is not set (because int() won't work on an empty string).  We could fit an if in there to handle that, but it looked pretty unwieldly so I went with the function.

Reply:
ah wait, no I see you're talking a different function than the one that was failing.  maybe environ gives something more tractable than getenv did.  didn't test that one.